R/as.vpts.R

```diff
@@ -35,13 +34,6 @@ as.vpts <- function(data) {
     msg = "`data` must contain data of a single radar."
   )
 
-  if (!exists("cached_schema")) {
```

Comment:
It seems you should call `validate_schema()` in this function as well, since I see no longer a validation of the schema in as.vpts(). That would be helpful for users trying to convert data.frames (that they may have manually edited) into a vpts object.

Reply:
Yes that is a good point